src/api/include/migraphx/migraphx.h

```diff
@@ -197,6 +203,43 @@ migraphx_status migraphx_parse_onnx_buffer(migraphx_program_t* out,
                                            size_t size,
                                            migraphx_onnx_options_t options);
 
+migraphx_status migraphx_op_names_destroy(migraphx_op_names_t op_names);
+
+migraphx_status migraphx_op_names_create(migraphx_op_names_t* op_names);
+
+migraphx_status migraphx_op_names_add(migraphx_op_names_t op_names, const char* name);
+
+migraphx_status migraphx_op_names_size(size_t* out, migraphx_op_names_t op_names);
+
+migraphx_status migraphx_op_names_get(const char** out, migraphx_op_names_t op_names, size_t idx);
```

Comment:
I dont think we need that `size` and `get` method.

Reply:
removed